web-frame/src/app-shared/core/errors/get-http-response-validation-errors.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * @license
+ * Copyright (c) 2018-2020 THREEANGLE SOFTWARE SOLUTIONS SRL
+ * Available under MIT license webFrame/LICENSE
+ */
+import { HttpErrorResponse } from '@angular/common/http';
+import { ValidationErrors } from '@angular/forms';
+
+import { HttpStatusCode } from '../interfaces/common.interface';
+
+import { isApiError } from './api-error.interface';
+
+export function getHttpResponseValidationErrors(response: HttpErrorResponse): ValidationErrors {
+  if (response.status === HttpStatusCode.CONNECTION_REFUSED) {
+    return {
+      'GENERAL.ERROR.CONNECTION_REFUSED': true,
+    };
+  }
+
+  // TODO: Add support for more multiple translations.
+  if (isApiError(response.error)) {
+    return {
+      [response.error.name]: true,
```

Comment:
I've added this TODO here in case we want to add more complex error responses. Currently, we only support a single translatable error message.

Reply:
Removed the todo and created #144